pkg/webui/console/components/webhook-form/index.js

```diff
@@ -228,11 +281,10 @@ export default class WebhookForm extends Component {
           <Form.SubTitle title={m.endpointSettings} />
           <Form.Field
             name="base_url"
+            type="button"
```

Comment:
Was this diff intentional ?

Reply:
Indeed this is a faulty change and should be reverted @ryaplots.

Reply:
I believe Bogdan fixed this fault within another PR (https://github.com/TheThingsNetwork/lorawan-stack/pull/4997). We disscussed it this morning